code/builders/builder-manager/src/utils/framework.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+
+export const rendererPathToName = (rendererPath?: string) => {
+  const value = rendererPath?.split(path.sep).pop()?.toLowerCase();
+
+  if (!value) {
+    return undefined;
+  }
+
+  if (value.includes('vue')) {
+    return 'vue';
+  }
+
+  return value;
+};
+
+export const normalizeBuilderName = (builderValue?: string | { name: string }) => {
```

Comment:
This should be refactored. Renderer, Framework, and Builder all can be represented as strings or as `{ name: string, options: any }`. AFAIK.

Reply:
What do you mean? As in I should make this more generic instead of just for the builder name?

Reply:
You do this logic for the builder but not for the renderer. You should do it for both. And refactor to keep it DRY

Reply:
I was under the impression that renderer only comes in as a string path which is why I didn't do the same thing

Reply:
We talked on Tuple and figured it out 😁